build.js

```diff
@@ -429,36 +437,49 @@ const getExtAttr = (node, name) => {
   return node.extAttrs && node.extAttrs.find((i) => i.name === name);
 };
 
-// https://webidl.spec.whatwg.org/#dfn-exposure-set
-const getExposureSet = (node) => {
-  // step 6-8
+// https://webidl.spec.whatwg.org/#Exposed
+const getExposureSet = (node, scopes) => {
+  // step 6-8 of https://webidl.spec.whatwg.org/#dfn-exposure-set
   const attr = getExtAttr(node, 'Exposed');
   if (!attr) {
     throw new Error(
       `Exposed extended attribute not found on ${node.type} ${node.name}`
     );
   }
-  const globals = new Set();
+  const exposure = new Set();
   switch (attr.rhs.type) {
     case 'identifier':
-      globals.add(attr.rhs.value);
+      exposure.add(attr.rhs.value.replace('GlobalScope', ''));
```

Comment:
Is `.replace('GlobalScope', '')` needed here? We should be processing an extended attribute like `[Exposed=Worker]` here, and the 'GlobalScope' suffix isn't part of the identifiers used in the extended attribute.

Reply:
This is to handle the same issue for `RTCIdentityProviderGlobalScope`.  Since this is an issue with the spec and we're special casing it during our search for `Globals`, we can special-case it here too!